.github/workflows/pr_author_auto_assign.yaml

```
@@ -0,0 +1,21 @@
+  pull-requests: write
+
+jobs:
+  assign:
+    name: PR author auto assign
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: bash
+    steps:
+    - uses: toshimaru/auto-author-assign@v2.1.1
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)